tests/model_regress/test_pickle.py

```diff
@@ -5,11 +5,14 @@
 import sys
 import warnings
 
+from os.path import abspath, dirname, split
```

Comment:
Could we reference these functions as `os.path.X` instead of importing them individually (for consistency with the rest of Django)?